README.md

```diff
@@ -91,4 +91,4 @@ You can then interact with the guardians and lightning gateway through the mproc
 If you want to run with UIs, see the [UI](https://github.com/fedimint/ui) repo for developer environment instructions.
 
 # Contributing
-For contribution guidelines, Areas of contributions and how to get involved, please refer to the [Contributing Guidelines](CONTRIBUTING.md).
\ No newline at end of file
+For contribution guidelines, Areas of contributions and how to get involved, please refer to the [Contributing Guidelines](CONTRIBUTING.md).
```

Comment:
wait what changed here

Reply:
I didn't change anything here.

Reply:
looks like new line missing on left, so it is fine

Reply:
Ah got it. 
I think we can merge this now.

Reply:
You're not trying to sneak-in some ssh backdoors in whitespace changes, are you? ;)

Reply:
Obviously not ser ;)